charts/icm-as/values.yaml

```diff
@@ -528,30 +528,29 @@ replication:
   # the deprecated "targetSystemUrl" / "sourceDatabaseName" configurations
   # Configuration for source system that participates in replication
   # source:
-    # the external URL of the webServer/proxy/ingress e.g. https://icm-web-edit-wa:443
-    # webserverUrl: <sourceExternalUrl>
-    # either mutual exclusive databaseUser or databaseLink have to be configured
-    # databaseLink: <sourceDatabaseLink>
-    # databaseUser: <sourceDatabaseUser>
-    # databaseName: <sourceDatabaseName>
+  # the external URL of the webServer/proxy/ingress e.g. https://icm-web-edit-wa:443
```

Comment:
Please revert the indentation changes for `source` and `targets`

Reply:
Uh Oh, that was an unnoticed autoformatting change of VSCode. I'd rather see that indendation inside the comments, if possible.
But yes, definitely not in that PR